tests/integration/source_code/solacc.py

```diff
@@ -175,7 +175,7 @@ def _lint_dir(solacc: _SolaccContext, soldir: Path):
     linted_files = set(files_to_skip)
     # lint solution
     start_timestamp = datetime.now(timezone.utc)
-    advices = list(ctx.local_code_linter.lint_path(soldir, linted_files))
+    advices = list(ctx.local_code_linter.lint_path(soldir))
```

Comment:
This needs to be addressed. An option would be to extend the `FolderLoader` with an `exclude_paths` parameter, then pass the `files_to_skip` to the `FolderLoader`. It would also be a speed up as the files while not be read. We could als use such a field in the `WorkspaceConfig` to exclude certain folders from being processed